code/go/0chain.net/smartcontract/storagesc/challenge.go

```diff
@@ -451,7 +541,7 @@ func (sc *StorageSmartContract) verifyChallenge(t *transaction.Transaction,
 	}
 
 	var (
-		threshold = len(challReq.Validators) / 2
+		threshold = challReq.TotalValidators / 2
 		pass      = success > threshold ||
 			(success > failure && success+failure < threshold)
```

Comment:
Mark for later  discussion about the `pass` calculation and the following partial success reward

Reply:
created an issue for the same: https://github.com/0chain/0chain/issues/1128